pkg/create/templates/scripts.go

```diff
@@ -20,11 +20,11 @@ fi
 #
 if [ "$(ls /tmp/artifacts/ 2>/dev/null)" ]; then
   echo "---> Restoring build artifacts..."
-  mv /tmp/artifacts/. ./
+  mv /tmp/artifacts/* ./
 fi
 
 echo "---> Installing application source..."
-cp -Rf /tmp/src/. ./
+cp -Rf /tmp/src/* ./
```

Comment:
@bparees You mean the folder .s2i. I misunderstand the script. I wander to konw why the code generates the hidden .s2i,? Once used s2i create ... command,, still need to modify files (script, run etc.) in folder .s2i Why not make it visible?